src/code42cli/cmds/devices.py

```diff
@@ -466,8 +471,9 @@ def _break_backup_usage_into_total_storage(backup_usage):
     total_storage = 0
     archive_count = 0
     for archive in backup_usage:
-        archive_count += 1
-        total_storage += archive["archiveBytes"]
+        if archive["archiveFormat"] == "ARCHIVE_V1":
```

Comment:
This should be `if archive["archiveFormat"] != "ARCHIVE_V2":`, since while V3 archives are not prevalent, they still should show up in the results.